src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -111,16 +111,17 @@
 Cannot_Create_ClusterRoleBindings_Fault_Type = 'Cannot create cluster role bindings on this Kubernets cluster'
 CC_Provider_Namespace_Not_Registered_Fault_Type = "Connected Cluster Provider MS.K8 namespace not registered"
 Default_Namespace_Does_Not_Exist_Fault_Type = "The default namespace defined in the kubeconfig doesn't exist on the kubernetes cluster."
-CLIENT_PROXY_VERSION = '1.1.0'
+ClusterConnect_Not_Present_Fault_Type = "Cluster connect feature not present in the chosen cloud."
```

Comment:
pls change to "cluster-connect-feature-unavailable"

Reply:
done